index.html

```diff
@@ -1,8 +1,118 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
   </head>
-
   <body>
+    <section class="background">
+      <header>
+        <ul>
+          <li>central pennsylvania</li>
+          <li>open source conference</li>
```

Comment:
I understand the idea of using li, but I don't see the bigger picture of using it can you explain?

Reply:
Idk its the first idea that popped in my head, i also couldve used flex then column it but we cant use flex

Reply:
what did you do?

Reply:
i guess you dont really need anything

Reply:
With it being such a small change I feel it will be best if we keep it and experiment a little. It can be an easy way to do the title if it works out.